Update STRING_VALUE column type to TEXT in AlgorithmParam schema to s…

weili-apidb opened the pull request:
…upport larger values

Commit:
Update STRING_VALUE column type to TEXT in AlgorithmParam schema to support larger values

mvnural commented on Definition/config/gus_schema.xml:
```diff
@@ -108,7 +108,7 @@
             <column id="Core/AlgorithmParam/ALGORITHM_PARAM_ID" name="ALGORITHM_PARAM_ID" nullable="false" length="10" precision="0" type="NUMBER"/>
             <column id="Core/AlgorithmParam/ALGORITHM_PARAM_KEY_ID" name="ALGORITHM_PARAM_KEY_ID" nullable="false" length="5" precision="0" type="NUMBER"/>
-            <column id="Core/AlgorithmParam/STRING_VALUE" name="STRING_VALUE" nullable="true" length="3000" precision="0" type="STRING"/>
+            <column id="Core/AlgorithmParam/STRING_VALUE" name="STRING_VALUE" nullable="true" type="TEXT"/>
```
GUS installer expects either a STRING or CLOB in the XML definition. The postgres installer converts it to TEXT during schema installation. Also they are not relevant for the CLOB field but I think we still need the length & precision attributes present. You may just put  `length="0" precision="0" `

Other than these, I don't see any problems making this a TEXT field instead VARCHAR. I'll merge it if @jbrestel says OK as well.

mvnural committed:
Fix syntax in previous commit.

GusSchema expects a CLOB value here. The installer will translate this to TEXT for postgres.
